examples/features/detached/detached_3_torch_distributed.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# To run:
+# python -m torch.distributed.run --nnodes=1 --nproc_per_node=2 \
+#   --master_addr 127.0.0.1 --master_port 29400 --max_restarts 0 \
+#   detached_3_torch_distributed.py
+
+import logging
+import random
+
+import determined as det
+from determined.experimental import Determined
```

Comment:
```python
from determined.experimental import client
...
client.Determined
```

Reply:
oh wait, I see how you're using it, you can just do

```python
from determined.experimental import client
```

And use the `client` module as you were using `Determined()`.